packages/web/package.json

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
im not sure if `web` is actually the right name for this because its not for the web, its for the builder only. I think a better name is `builder` to be specific for anyone else

Reply:
Just name it web since the package published is web, I don't have any particular preference rather than keep consistency with the names published on NPM/Github repo.

Reply:
I agree that even if its the wrong name we already chose it a while ago and the costs of changing it now >>>> costs of having a slightly inaccurate name.